dev_guide/builds/jenkins_pipeline_builds.adoc

```diff
@@ -0,0 +1,169 @@
+|   A Pipeline build is triggered  |
+|   in OpenShift via command line, |--------------------------------+
+|   web console, or API            |                                |
+|                                  |                                |
++----------------------------------+                                |
+                                                                    v
+                                     +------------------------------------------------------------+
+                                     |                                                            |
+                                     |   The Jenkins Master for your project observes that a new  |
+                +--------------------|   Pipeline build object has been created. A new Pipeline   |
+                |                    |   job is then created and a new Jenkins Slave is requested |
+                |                    |                                                            |
+                |                    +------------------------------------------------------------+
+                v
++------------------------------------------------------------+
+|                                                            |
+|   The content of the JenkinsFile is run on the Jenkins     |
+|   Slave pod. A new build on OpenShift is triggered via     |---------------------+
+|   The "Trigger a Build" command in the JenkinsFile         |                     |
+|                                                            |                     |
++------------------------------------------------------------+                     |
+                                                                                   v
+                                                                  +---------------------------------+
+                                                                  |                                 |
+                                                                  |   OpenShift runs the build as   |
+                +-------------------------------------------------|   normal and marks the Pipeline |
+                |                                                 |   build as complete             |
+                |                                                 |                                 |
+                |                                                 +---------------------------------+
+                v
++---------------------------------+
+|                                 |
+|   The Jenkins Master observes   |
+|   that the Pipeline build is    |
+|   complete and removes the      |
+|   Jenkins Slave pod             |
+|                                 |
++---------------------------------+
+----
+
+[[openshift-jenkins-Pipeline-plugin]]
+
+== OpenShift Jenkins Pipeline Plugin
+
+The https://github.com/openshift/jenkins-client-plugin[OpenShift Jenkins Pipeline Plugin] must be installed
```

Comment:
openshift client plugin  or openshift jenkins client plugin.

Reply:
updated in next push